### PR TITLE
Sketched out landing page

### PR DESCRIPTION
**Pull request by IslaMcN**
Until I receive feedback or more information about how we want layout, this is what I have.

### CODE REVIEW

File: less/footer.less [hunk 1]
```diff
@@ -0,0 +1,30 @@
+footer{
+    margin: 2%;
+    padding: 5%;
+    
+    ul{
+        li{
+            list-style: none;
```
Comment (dswhitely1) [inline less/footer.less]: I do believe that you only need to specify the `list-style: none;` in the `ul` tag.

File: less/variables.less [hunk 2]
```diff
@@ -0,0 +1 @@
+@mobileview: ~"(Max-width: 800px)";
```
Comment (dswhitely1) [inline less/variables.less]: Remember for MVP, you have to have a Mobile view looking professional at 500px, and a desktop view looking professional at 1000px.  For Stretch, you can incorporate a tablet view at 800px as well.

File: less/footer.less (hunk 1 above)
Comment (IslaMcN) [reply less/footer.less]: Ah thank you I'll try that out.

File: less/variables.less (hunk 2 above)
Comment (IslaMcN) [reply less/variables.less]: I must have read that wrong. I'll fix that right now, thank you!

**Pull request merged**